app/app/src/main/res/layout/content_joystick.xml

```diff
@@ -5,6 +5,15 @@
     android:layout_height="fill_parent"
     android:visibility="gone" >
 
+
+    <com.example.myapplication.JoyStick
+        android:background="#EBEBEB"
+        android:id="@+id/joy1"
+        android:layout_width="300dp"
+        android:layout_height="300dp"
```

Comment:
What would it look like if it was match-parent or fill-parent instead?

Reply:
It will be much bigger and filled with all the screen below headers, now is just little bigger than the start button.